crates/hc_deepkey_sdk/README.md

```diff
@@ -0,0 +1,11 @@
+[![](https://img.shields.io/crates/v/hc_deepkey_sdk?style=flat-square)](https://crates.io/crates/hc_deepkey_sdk)
+
+See source code [github.com/holochain/deepkey](https://github.com/holochain/deepkey)
```

Comment:
Link no longer accurate, and neither is the badge

Reply:
How is the badge not accurate?

Reply:
Good point, it's going straight to crates.io. Read to quickly!